src/components/organisms/EntityDetails/EntityDetailsContent/EntityDetailsContent.tsx

```diff
@@ -97,14 +97,18 @@ const EntityDetailsContent: React.FC = () => {
           type: RunningContextType.userUI,
         },
       },
-    }).then(res => {
-      displayDefaultNotificationFlow(res, () => {
-        analyticsTrack('trackEvents', {
-          type,
-          uiEvent: `run-${entity}`,
+    })
+      .then(res => {
+        displayDefaultNotificationFlow(res, () => {
```

Comment:
```suggestion
        return displayDefaultNotificationFlow(res, () => {
```
and same for `runEntity`